### ISSUE TITLE
Double check that we have permission to use all images and have included accreditation where approprieate

### ISSUE THREAD

**Issue by joelostblom** [UBC-MDS/ds-toolbox#5]

**Comment by joelostblom** [comment]
Done for module 1 and 2 when #100 is merged

**Comment by armanawn** [comment]
Done for module 6, 7, and 8 when #100 is merged.

**Comment by flor14** [comment]
@joelostblom here is my research about the copyright of the images (#5). There is only one image that could be problematic, but let me know your thoughts.

These are the images I have checked for copyright. Remember that all of them are included in the videos. 

* Allison Horst image (module5_05 slides)
She suggests donating to [indigenous woman who hikes](https://github.com/allisonhorst/stats-illustrations#is-the-artwork-useful-for-your-teaching-materials-i-would-be-so-grateful-if-you-would). I think that it could be a nice gesture to donate something. Also, she wants alt text in the slide, I added it.

* PHD comics (module3_01 slides) https://phdcomics.com/about.php
This is the most complicated because we should send an email
<img width="413" alt="image" src="https://user-images.githubusercontent.com/29610858/194719495-aed8dd7b-6843-4c4c-acbc-25279af36de3.png">

* XKCD comics (module3_04 slides)
https://xkcd.com/1296/
This work is licensed under a [Creative Commons Attribution-NonCommercial 2.5 License](https://creativecommons.org/licenses/by-nc/2.5/).

* The Turing Way (module3_01 slides)
I cited the image as it is mentioned [here](https://zenodo.org/record/3332808#.Y0GvQ1LMJwo). It is a CC-BY license.

**Comment by joelostblom** [comment]
@flor14 

> She suggests donating to [indigenous woman who hikes](https://github.com/allisonhorst/stats-illustrations#is-the-artwork-useful-for-your-teaching-materials-i-would-be-so-grateful-if-you-would). I think that it could be a nice gesture to donate something. Also, she wants alt text in the slide, I added it.

Agree, could you ask tiff/varada for a small amount in the channel? Or would you prefer if I ask?

> PHD comics (module3_01 slides) https://phdcomics.com/about.php
This is the most complicated because we should send an email

Agree, could you send them the email?

The other two looks good already, thanks for adding the info to the slides!

**Comment by flor14** [comment]
Done @joelostblom . Let's wait for an answer before closing this.
